packages/xo-web/src/xo-app/tasks/index.js

```diff
@@ -151,6 +153,53 @@ const FINISHED_TASKS_COLUMNS = [
   },
 ]
 
+const XO_TASKS_COLUMNS = [
+  {
+    itemRenderer: task => task.name,
+    name: _('name'),
+  },
+  {
+    itemRenderer: task => (task.objectId === undefined ? null : renderXoItemFromId(task.objectId, { link: true })),
+    name: _('object'),
+  },
+  {
+    itemRenderer: task => {
+      const progress = task.properties?.progress
+
+      return progress === undefined ? null : (
+        <progress style={TASK_ITEM_STYLE} className='progress' value={progress * 100} max='100' />
```

Comment:
After discussing with @fbeauchamp , we decided that progress should simply be a number between `0` and `100`.

```suggestion
        <progress style={TASK_ITEM_STYLE} className='progress' value={progress} max='100' />
```